sup3r/postprocessing/collection.py

```diff
@@ -488,12 +490,16 @@ def _collect_flist(self, feature, masked_meta, time_index, shape,
             to be collected.
         out_file : str
             File path of final output file.
-        target_final_meta : str
-            Target final meta containing coordinates to keep from the
-            full file list collected meta
+        target_final_meta : pd.DataFrame
+            Meta data containing coordinates to keep from the full file list
+            collected meta. This can be but is not necessarily a subset of the
+            full list of coordinates for all files in the file list. This is
+            used to remove coordinates from the full file list which are not
+            present in the target_final_meta.
         masked_target_meta : pd.DataFrame
-            Collected meta data with mask applied from target_final_meta so
-            original gids are preserved.
+            Dataframe containing the same coordinates contrained in the
+            target_final_meta but with the same gids that are present in the
+            full list of coordinates present in the full file list.
```

Comment:
Lol yeah it is confusing. the masked_meta is the same as masked_target_meta but for the subset of files provided. masked_target_meta can be derived from the unmasked meta for the full file list + the final target meta (its just the subset of the full meta with coordinates present in the final target).The only reason target_final_meta is passed here is because of the initialization check for the output file. Which could be removed.